openeo/rest/datacube.py

```diff
@@ -711,6 +725,10 @@ def merge(self, other: 'DataCube', overlap_resolver: PGNode = None) -> 'DataCube
             'cube2': {'from_node': other._pg},
         }
         if overlap_resolver:
+            # TODO: for 1.0.0 support
```

Comment:
Can you add `#125`  in this TODO line? makes it easier to find 1.0 related TODOs

Reply:
or wait: as far as I understand the spec, there is no need for additional `process` level here. I think this is correct:
```
...
    "overlap_resolver": {
        "process_graph": {
            "max": { "process_id": "max" ....
```

Reply:
Yes you are right, I removed the comments